resourcegroupstaggingapi/resourcegroupstaggingapi.go

```diff
@@ -0,0 +1,127 @@
+package resourcegroupstaggingapi
```

Comment:
You should probably use the common library instead...

https://github.com/YaleSpinup/aws-go/tree/master/services/resourcegroupstaggingapi

Reply:
you can extend it if you have more filters, etc